photon-core/src/test/java/org/photonvision/common/configuration/ConfigTest.java

```diff
@@ -141,4 +142,18 @@ public void testJacksonHandlesOldVersions() throws IOException {
 
         new File("test.json").delete();
     }
+
+    @Test
+    public void testJacksonHandlesOldTargetEnum() throws IOException {
+        var str =
+                "[ \"AprilTagPipelineSettings\", {\n" + "  \"targetModel\" : \"k6in_16h5\"\n" + "} ]\n";
+        var writer = new FileWriter("test.json");
```

Comment:
Can you change this to Files.makeTempFile or something?

Reply:
There is another test in this file that I copied and this is how they did it. I'll make the change.

Reply:
This function, and the one before it now use temp files.